README.md

```diff
@@ -14,42 +14,25 @@ All information can be found on [our blog at hello.iRail.be](http://hello.irail.
 
 _note: you'll also need to have [nodejs](https://nodejs.org), [composer](http://getcomposer.org) and PHP curl extension installed on your system_
 
- * Step 1: clone this repo
+ * Step 1: Clone this repo
  * Step 2: `composer install`
- * Step 3: make sure storage is writable: `chmod 777 storage`
+ * Step 3: Make sure storage is writable: `chmod 777 storage`
  * Step 4: Run your test server: `php -S localhost:8008 -t api`
  * Step 5: Enjoy your own iRail API at http://localhost:8008/connections.php?from=Gent%20Sint%20Pieters&to=Antwerp
 
-## Update stations list ##
-
-Stations are updated through the irail/stations composer package. Just perform a `composer update` in the root of the project
-
-## Integrating train occupancy scores ##
-
-We use user feedback and our own knowledge to tell users how busy a train is thank to the crowdfunding campaign of [Spitsgids](https://spitsgids.be).
-
-Optionally, you can thus also set up the occupancy scores on your local machine, but this is a bit more complicated as you will also need mongodb installed:
+**Optional**, if you want to set up the iRail API with occupancy scores you will need to set up a MongoDB database:
 
- * MongoDB: [installation instructions](https://www.mongodb.com/download-center?jmp=nav#community)
- * MongoDB module for PHP, installable through pecl or your systems package manager
+ * Step 6: Install MongoDB: `brew install mongodb`
+ * Step 7: Install the MongoDB module for PHP: `pecl install mongodb`
+ * Step 8: Include MongoDB: `composer require mongodb/mongodb:^1.0` (make sure to not commit the composer.json file)
+ * Step 9: Import the data (the structural.csv file) in MongoDB that already exists in the [Spitsgids-data repo](https://github.com/osoc16/Spitsgids-data): `mongoimport -d irail -c structural --type csv --file structural.csv --headerline`
+ * Step 10: Add MongoDB environment variables: `cp .env.example .env` (If your MongoDB URL is different or you want another database name you can change this file)
+ * Step 11: Run MongoDB: `mongod`
```

Comment:
Why? If it's installed, it should run in the background already? Or people should know themself that this should be running, no? I'd remove this step